client/src/components/products/AddProduct.js

```diff
@@ -69,7 +74,7 @@ export default class AddProduct extends Component {
           category: [],
```

Comment:
this needs to be changed to empty string : ""
as it is in the model

Reply:
True that. Thanks for the review, I updated the PR. 🎈

Reply:
super!!